### PR TITLE
ContactUs page Fixed #24

### LINKED ISSUE #24: Create contact us page
Create contact us page and improve the design.
Screen Shot 2021-03-23 at 5 46 34 PM

### PR DESCRIPTION
**Pull request by JalalArif**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import InfoSection from "./components/InfoSection/InfoSection"
-
+import ContactUs from './components/ContactUs/contactUs'
 function App() {
 	return (
 		<div>
-			<InfoSection/>
+			<ContactUs/>
```
Comment (murtaja1) [inline src/App.js:5]: remove <ContactUs/> from app.js

File: src/components/ContactUs/contactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {Button,CssBaseline,TextField,Grid,Box,Typography,makeStyles,Container} from '@material-ui/core';
+
+
+const useStyles = makeStyles((theme) => ({
```
Comment (murtaja1) [inline src/components/ContactUs/contactUs.jsx]: create a different file for the styling.

File: src/App.js (hunk 1 above)
Comment (murtaja1) [inline src/App.js]: remove from contactus app.js

**Review by murtaja1 — CHANGES_REQUESTED** [review]

**Comment by maqalaqil** [comment]
@JalalArif please check comments

File: src/components/ContactUs/contactUs.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    margin: theme.spacing(3, 0, 2),
+    backgroundColor:"#3AAFA9",
+    color:"white"
+  },
+}));
+
+export default function contactUs() {
+  const classes = useStyles();
+
+  return (
```
Comment (Othmanosx) [inline src/components/ContactUs/contactUs.jsx:27]: Add Navbar component to the page

File: src/components/ContactUs/contactUs.jsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+          <Grid container justify="flex-end">
+            <Grid item>
+             
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+      <Box mt={5}>
+      </Box>
+    </Container>
```
Comment (Othmanosx) [inline src/components/ContactUs/contactUs.jsx:107]: Add footer component to the page

**Comment by Othmanosx** [comment]
Resolve App.js conflict

File: src/App.js (hunk 1 above)
Comment (Othmanosx) [reply src/App.js]: don't add anything in app.js

**Review by Othmanosx — COMMENTED** [review]
keep app.js blank

File: src/components/ContactUs/contactUs.jsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+export default function ContactUs() {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+       
+        <Typography component="h1" variant="h5">
```
Comment (ZhinAbubakr) [inline src/components/ContactUs/contactUs.jsx:14]: center align the text inside the h4 tag

**Review by ZhinAbubakr — CHANGES_REQUESTED** [review]

**Review by ZhinAbubakr — APPROVED** [review]

**Review by murtaja1 — APPROVED** [review]

**Review by maqalaqil — APPROVED** [review]
LGTM

**Review by Ravyar19 — APPROVED** [review]

**Pull request merged**